lib/jsonapi/include_directive.rb

```diff
@@ -68,5 +70,26 @@ def to_string
 
       string_array.join(',')
     end
+
+    class InvalidKey < StandardError; end
+
+    private
+
+    def validate(key)
+      return if valid?(key)
+
+      raise InvalidKey, key
+    end
+
+    def valid?(key)
+      key.match(valid_json_key_name_regex)
+    end
+
+    def valid_json_key_name_regex
+      # not start with hyphen/underscore/space
+      # not end with hyphen/underscore/space
+      # contains a-zA-Z, *  and hyphen/underscore/space in member names
+      /^(?![\s\-_])[\w\s\-\*]+(?<![\s\-_])$/
```

Comment:
This regex does not seem correct (numbers for instance are allowed in member names).